Dockerfile

```diff
@@ -0,0 +1,52 @@
+# Use Ubuntu 14.04 as the base container
+FROM ubuntu:14.04
+
+MAINTAINER "Robert McDermott" robert.c.mcdermott@gmail.com
+
+# ADD the CRAN Repo to the apt sources list
+RUN echo "deb http://cran.fhcrc.org/bin/linux/ubuntu trusty/" > /etc/apt/sources.list.d/cran.fhcrc.org.list
+# Add the package verification key
+RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 51716619E084DAB9
+
+# Update the system and install packages
+RUN apt-get -y update && apt-get -y install \
+	r-base=3.2.2* \
+	vim \
+	make \
+	m4 \
+	gcc \
+	g++ \
+	libxml2 \
+	libxml2-dev \
+        python-pip && \
+        pip install websocket-client 
+
+# create the sttrweb user and data directory
+RUN useradd -u 7534 -m -d /home/sttrweb -c "sttr web application" sttrweb && \
```

Comment:
@LisaMc if folks deploy Oncoscape via Docker then they don't need to worry about the directory structure inside the container; it has no relation to the system they are running the container on. As long as they have a Docker capable server, desktop or laptop (could be Windows, Linux or Mac) they can think of the Oncoscape container as a single large binary executable.

If folks want to provide their own datasets and libraries they can mount part of the host running Docker inside the container when the container is launched with a run time argument.